tests/EventTest.php

```diff
@@ -253,6 +254,18 @@ public function gettersAndSettersDataProvider(): array
         ];
     }
 
+    public function testBacktraceSetter(): void
+    {
+        $stacktrace = $this->createMock(Stacktrace::class);
+        $event = new Event();
```

Comment:
Please add a blank line above this one to improve readability

Reply:
done and amended